CapMan.Raylib/ScoreRenderer.cs

```diff
@@ -0,0 +1,13 @@
+using Raylib_cs;
+public class ScoreRenderer
+{
+    public const int BlockedHeight = 30;
+    public void Render(Game game, int top, int left)
+    {
+        var scoreLength = game.Score.ToString().Length * 10;
+        Rectangle container = new(left, top, 100 + scoreLength, BlockedHeight);
+
+        Raylib.DrawRectangleLinesEx(container, 2, Color.Yellow);
+        Raylib.DrawText($"Score : {game.Score}", left + 5, top + 5, BlockedHeight - 10, Color.Yellow);
+    }
+}
```

Comment:
Nice and simple! A few notes:

* You can use `Raylib.MeasureText(text, font-size);` to get the width of text rather than using the length of the string.

* I would prefer a `Padding` const rather than using ` + 5`.

* I would prefer a `FontSize => BlockedHeight - 30`

* I think this should be in `namespace CapMan.Raylib;`

Reply:
It also might be nice to use a custom font that is a little more pixelated: https://www.raylib.com/examples/text/loader.html?name=text_raylib_fonts